### DISCUSSION TITLE
On Kibana/Elastic, send alert emails with rules

### DISCUSSION THREAD

**Discussion by ghost** [telekom-security/tpotce#932] [Q&A] [answered]
Hello T-pot and Telekom Security communities,

I have the last version 20.6.
On Kibana/Elastic, I want to send emails when one of the Honeypot find an attack.

So, I went here: _MyIP:64297/kibana/app/management/insightsAndAlerting/triggersActions/connectors_
But I have this error : _Encrypted saved objects are not available. To create a rule, set a value for xpack.encryptedSavedObjects.encryptionKey in your kibana.yml file and ensure the Encrypted Saved Objects plugin is enable_
![Capture](https://user-images.githubusercontent.com/94058037/141123049-1d8ef21e-042f-4b16-84cd-23372b889e4d.JPG)

I tried a lot of things in _/opt/tpot/etc/tpot.yml_ but without success.

Can you help me ?

Thanks and have a good day !

**Comment by ghost** [comment]
Hello,

@t3chn0m4g3 can you help me?
I try everything but nothing works...

Thanks for your help and have a good day

**Comment by t3chn0m4g3** [comment, marked as answer]
We do not support encryption or alerting with this setup of Elastic.

**Comment by ghost** [comment]
One other solution is to install jertel/elastalert2 docker container to send emails
